src/CppUTestExt/MockSupport_c.cpp

```diff
@@ -382,7 +384,7 @@ MockExpectedCall_c* withConstPointerParameters_c(const char* name, const void* v
 
 MockExpectedCall_c* withFunctionPointerParameters_c(const char* name, void (*value)())
 {
-    expectedCall = &expectedCall->withParameter(name, value);
+    expectedCall = &expectedCall->withParameter(name, (Cpp_Func*)value);
```

Comment:
I doubt anyone will remove the casts. It is so weird that he would at least check the history. And besides, you are checking all commits anyway :P